### ISSUE TITLE
Provided ton network config is wrong

### ISSUE THREAD

**Issue by X1ting** [broxus/ton-indexer#3]
```
-> curl -XGET https://raw.githubusercontent.com/tonlabs/net.ton.dev/master/configs/net.ton.dev/ton-global.config.json
404: Not Found%
```
Could you update that file, or let me know where I can find correct config?

**Comment by Rexagon** [comment]
Thanks! However, note that this config is for the Everscale devnet network.

If you need mainnet global config, you should use this one - https://raw.githubusercontent.com/tonlabs/main.ton.dev/master/configs/ton-global.config.json

**Comment by X1ting** [comment]
Oh, I see, but both configs are for Ton network? Or it's somehow compatible with Venom network? Sorry for silly questions, I'm pretty newbie in crypto

**Comment by Rexagon** [comment]
This library is compatible with Everscale and Venom network, ton in the naming is a legacy.

There is no public global config for the Venom yet, but there will be a dedicated hackathon devnet soon with public global config.

**Comment by X1ting** [comment]
@Rexagon got you, thanks for explanation! So, mainly API (or how it correctly say for blockchain node), it the same for Everscale and Venom?  Can I test some basic interaction with blockchain node, based on Everscale, and port it later to Venom?

**Comment by Rexagon** [comment]
Yes, API is the same, but there are some differences in enabled features between these networks  (e.g. everscale devnet has a different external messages delivery protocol being tested).

Sure, you can test it in any working network. Please also note that this repo is not the whole node, but rather a library which can be used to write your own node to work with the blockchain on the lowest level (e.g. to be used in something like this https://github.com/broxus/everscale-monitoring). So you probably don't need to go this far if you are going to only work with smart contracts.

However, if you really want to do something low-level, then I'm very glad that someone else will use this and you can contact me in tg for any help :)

With the advent of Venom, the dev tools ecosystem should come into a more unified form, but now it's a mess.

**Comment by X1ting** [comment]
Got you, thanks a lot for explanation, appreciate it! Actually I'm not aiming to interact with smart contract, but I don't really know, how "low-level" is my task :D, so, connect with you in TG might help.
